content/en/docs/reference/glossary/dockershim.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Dockershim
+id: dockershim
+date: 2022-04-15
+full_link: /blog/2022/02/17/dockershim-faq/
+short_description: >
+  Dockershim is a component of Kubernetes v1.23 and earlier, which allows Kubernetes system components to communicate with Docker Engine.
+
+aka:
+tags:
+- fundamental
+---
+Dockershim is a component of Kubernetes version 1.23 and earlier. It allowed Kubernetes system components like kubelet to communicate with {{< glossary_tooltip text="Docker Engine" term_id="docker" >}}.
```

Comment:
As this is targeting the _main_ branch, we should write something like:
```suggestion
The dockershim is a component of Kubernetes version 1.23 and earlier. It allows the kubelet
to communicate with {{< glossary_tooltip text="Docker Engine" term_id="docker" >}}.
```

and then file an issue to update this text for the v1.24 release. We should also backport that change to the v1.23 docs after the v1.24 release has happened.